common/alerts.js

```diff
@@ -52,5 +52,21 @@
                 };
             }
         };
-    }]);
+    }])
+    .directive('compile', ['$compile', function ($compile) {
```

Comment:
We usually define these utility components in the `common/utils.js`. I think we should move this directive to there. If you can add a few comments about how it works would be good too. Most of our directives and components lack documentation which just makes it harder to figure out what they're supposed to do. We're not using jsdoc in chaise, but you can use the same syntax.

```javascript

/**
* THE DOCUMENTATION GOES HERE
*/
.directive('compile', ['$compile', function ($compile) {
...
})
```